Apps/Src/Current.c

```diff
@@ -107,10 +107,27 @@ void Task_AmperesMonitor(void *p_arg) {
 
     OS_ERR err;
 
+	bool isfirst_amperes_check = false;
```

Comment:
Maybe it would be better to rename this to something like `amperesHasBeenChecked`? Right now, the name and what it's being used for seem a bit backwards to me.

Reply:
Sounds gud